### PR TITLE
8.0 - account_invoice_entry_date - Invoice sequence based on registration_date

### PR DESCRIPTION
**Pull request by OpenCode**
When an in invoice or refund has a registration date, the invoice sequence must be based on registration date year

### CODE REVIEW

File: account_invoice_entry_date/models/account.py [hunk 1]
```diff
@@ -41,16 +41,37 @@ class AccountInvoice(models.Model):
...
+                ('date_stop', '>=', date_stop),
+                ('company_id', '=', invoice.company_id.id),
+                ('special', '!=', True),
+                ])
+        if period_ids:
+            period_id = period_ids[0]
+        else:
+            period_id = False
+        return period_id, date_start, date_stop
+
```
Comment (eLBati) [inline account_invoice_entry_date/models/account.py]: You can also use ```find``` method of ```account.period```

**Comment by scigghia** [comment]
👍

File: account_invoice_entry_date/models/account.py (hunk 1 above)
Comment (OpenCode) [inline account_invoice_entry_date/models/account.py]: find return id. I need id of period and start and stop date, too

**Comment by eLBati** [comment]
@OpenCode puoi fare lo squash dei 3 commit?

**Comment by OpenCode** [comment]
@eLBati scusa il ritardo. Squash effettuato. Travis verde.

**Pull request merged**